filetime_from_git/git_wrapper.py

```diff
@@ -2,7 +2,7 @@
 """
 Wrap python git interface for compatibility with older/newer version
 """
-import itertools
+from future.moves.itertools import zip_longest
```

Comment:
Good points, I've updated the fix accordingly. Also verified that it works with both versions.